### PR TITLE
Update to Sylius v1.2.0-BETA

### LINKED ISSUE #229: Symfony 4 support for v1.2.x
See related project https://github.com/Sylius/Sylius/projects/4.

### PR DESCRIPTION
**Pull request by pamil**
Closes #229.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     },
     "config": {
         "platform": {
-            "php": "7.1"
+            "php": "7.1.18"
```
Comment (teohhanhui) [inline composer.json]: Why?

Comment (pamil) [reply composer.json]: PHP 7.1.0 was not enough to install Symfony 4.1 IIRC

Comment (teohhanhui) [reply composer.json]: If it's not listed in Symfony's `composer.json`, I'd think it's not important enough. And we shouldn't set the platform this way anyway. It means we risk generating a `composer.lock` that only works with the very latest patch release of PHP 4.1, which I'm sure is not your intention.

Comment (pamil) [reply composer.json]: Good catch, I'll make an issue for that.

File: composer.json [hunk 2]
```diff
@@ -21,29 +21,29 @@
     ],
     "require": {
         "php": "^7.1",
-        "sylius/sylius": "1.2.*@dev"
+
```
Comment (teohhanhui) [inline composer.json]: Can we please stop adding this useless blank line? :rofl:

**Pull request merged**